src/ng/directive/attrs.js

```diff
@@ -422,6 +422,10 @@ forEach(['src', 'srcset', 'href'], function(attrName) {
           propName = null;
         }
 
+        // We need to sanitize the url at least once, in case it is a constant
+        // non-interpolated attribute.
+        attr.$set(normalized, $sce.getTrustedMediaUrl(attr[normalized]));
```

Comment:
Maybe we could check the current value and only call `$set()` if `getTrustedMediaUrl()` return a different value.